ngraph/frontend/onnx_editor/src/editor.cpp

```diff
@@ -330,6 +326,12 @@ std::string onnx_editor::ONNXModelEditor::model_string() const
     return m_pimpl->m_model_proto.SerializeAsString();
 }
 
+std::shared_ptr<Function> onnx_editor::ONNXModelEditor::get_function() const
+{
+    std::istringstream model_stream(model_string());
```

Comment:
@postrational ONNX Importer API doesn't expose a method that takes model proto. Is ok to use string representation here?